sfr_eff.c

```diff
@@ -375,9 +375,10 @@ cooling_direct(int i) {
 
     double ne = SPHP(i).Ne;	/* electron abundance (gives ionization state and mean molecular weight) */
 
-    double unew = DMAX(All.MinEgySpec,
-            (SPHP(i).Entropy + SPHP(i).DtEntropy * dloga) /
-            GAMMA_MINUS1 * pow(SPHP(i).EOMDensity * All.cf.a3inv, GAMMA_MINUS1));
+    /*Entropy at kick time after this timestep: note this uses this timestep's DtEntropy!*/
+    const double KickEntropy = SPHP(i).Entropy + SPHP(i).DtEntropy * dloga/2;
```

Comment:
There wasn't a /2 on the left hand side.

Reply:
But that is because the Entropy was formerly at the Kick time, ie, t - dt/2 and is now at drift time, t.